tensorrt_llm/_torch/cute_dsl_kernels/mega_moe_nvfp4/non_clc_mixed_cga.py

```
@@ -0,0 +1,372 @@
+                or self.fallback_cluster_count < 0
+            ):
+                raise ValueError(
+                    "fallback_cluster_count must be a non-negative Python "
+                    "int when fallback_cluster_shape is present."
+                )
+
+            if any(
+                preferred_dimension % fallback_dimension != 0
+                for preferred_dimension, fallback_dimension in zip(
+                    self.preferred_cluster_shape, self.fallback_cluster_shape
+                )
```

Ruff (0.16.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)